modules/dicehub/extension/provider.go

```diff
@@ -63,13 +69,29 @@ func (p *provider) Init(ctx servicehub.Context) error {
 				}),
 			))
 	}
-	go func() {
-		err := p.extensionService.InitExtension(FilePath, false)
-		if err != nil {
-			panic(err)
-		}
-		logrus.Infoln("End init extension")
-	}()
+
+	var once sync.Once
+	p.InitExtensionElection.OnLeader(func(ctx context.Context) {
+		once.Do(func() {
+			var doneValue atomic.Value
```

Comment:
once already use atomic value to prevent concurrency invoke, is this `doneValue` needless?

cc @sfwn

Reply:
no need.